### PR TITLE
rework smtp credential using

### PR DESCRIPTION
**Pull request by BelehovEgor**
now i don't save it in message service. I send request to Company to get credentials, if i don't have it now, but I can consume request about updated smtp for getting new info

### CODE REVIEW

**Review by MariyaSin — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

File: src/MessageService.Broker/Helpers/BaseEmailSender.cs [hunk 1]
```diff
@@ -41,10 +80,10 @@ protected bool Send(DbEmail email)
             catch (Exception exc)
             {
                 _logger?.LogWarning(exc,
-                            "Errors while sending email to {to} with subject: {subject} and body: {body}. Email replaced to resend queue.",
-                            email.Receiver,
-                            email.Subject,
-                            email.Body);
+                    "Errors while sending email to {to} with subject: {subject} and body: {body}. Email replaced to resend queue.",
```
Comment (djino97) [inline src/MessageService.Broker/Helpers/BaseEmailSender.cs:83]: LogError

**Review by TIove — APPROVED** [review]

File: src/MessageService.Broker/Helpers/BaseEmailSender.cs [hunk 2]
```diff
@@ -9,29 +13,64 @@ namespace LT.DigitalOffice.MessageService.Broker.Helpers
...
+                    SmtpCredentials.Host = result.Body.Host;
+                    SmtpCredentials.Port = result.Body.Port;
+                    SmtpCredentials.Email = result.Body.Email;
+                    SmtpCredentials.Password = result.Body.Password;
+                    SmtpCredentials.EnableSsl = result.Body.EnableSsl;
+
+                    return true;
+                }
+
+                _logger.LogWarning(logMessage);
```
Comment (djino97) [inline src/MessageService.Broker/Helpers/BaseEmailSender.cs:39]: _logger?.LogWarning

File: src/MessageService.Broker/Helpers/BaseEmailSender.cs [hunk 3]
```diff
@@ -9,29 +13,64 @@ namespace LT.DigitalOffice.MessageService.Broker.Helpers
...
+
+                return false;
```
Comment (djino97) [inline src/MessageService.Broker/Helpers/BaseEmailSender.cs:41]: can remove this return

**Review by djino97 — APPROVED** [review]

File: src/MessageService.Broker/Helpers/BaseEmailSender.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/MessageService.Broker/Helpers/BaseEmailSender.cs:83]: fixed

File: src/MessageService.Broker/Helpers/BaseEmailSender.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/MessageService.Broker/Helpers/BaseEmailSender.cs:39]: fixed

File: src/MessageService.Broker/Helpers/BaseEmailSender.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/MessageService.Broker/Helpers/BaseEmailSender.cs:41]: fixed

**Pull request merged**